webapp/src/generalSlices/userSlice.test.js

```diff
@@ -42,6 +43,7 @@ describe('async thunks', () => {
   const initialState = {
     loading: null,
     bsmLoading: null,
+    loginMessage: "",
```

Comment:
Let's use that prettierrc plugin to format here and get these double quotes to conform to the single quite standard we've selected (ctrl+k + ctrl+d shortcut in vscode to format)

Reply:
Updated to use single quotes